package.json

```diff
@@ -28,6 +28,7 @@
     "email-prop-type": "^3.0.0",
     "font-awesome": "^4.7.0",
     "history": "^4.7.2",
+    "moment": "^2.24.0",
```

Comment:
I checked out this npm page - 32k dependents on moment. That's a widely used library...